src/libsync/propagateupload.cpp

```diff
@@ -894,5 +898,392 @@ void PropagateUploadFile::abortWithError(SyncFileItem::Status status, const QStr
     done(status, error);
 }
 
+void PropagateBundleQNAM::start()
+{
+    QHttpMultiPart *multiPart = new QHttpMultiPart(QHttpMultiPart::RelatedType);
+    multiPart->setBoundary("boundrary123");
```

Comment:
This looks wierd. I suppose this should not be hardcoded, but be a sufficiently long random string.